src/swsssdk/scripts/sonic-db-cli

```diff
@@ -33,6 +33,6 @@ else:
         if resp is None:
             print ""
         elif isinstance(resp, list):
-            print " ".join(resp)
```

Comment:
>resp [](start = 27, length = 4)

Did you compare the behaviors of sonic-db-cli/redis-cli on a key containing '\n' ?

Reply:
Looks the key cannot be input with '\n', I didn't find a way to input any key with '\n'

```
root@ASW-7005:/# redis-cli -n 4 HSET "TEST1\n" "new1\n" "line1\n"
(integer) 1
root@ASW-7005:/# redis-cli -n 4 HSET "TEST2\\n" "new2\\n" "line2\\n"
(integer) 1
root@ASW-7005:/# 
root@ASW-7005:/# 
root@ASW-7005:/# redis-cli -n 4 KEYS "*TEST*"                        
1) "TEST1\\n"
2) "TEST2\\n"
root@ASW-7005:/# redis-cli -n 4 HGETALL "TEST1\\n"
1) "new1\\n"
2) "line1\\n"
root@ASW-7005:/# redis-cli -n 4 HGETALL "TEST2\\n"
1) "new2\\n"
2) "line2\\n"
```

Reply:
`redis-cli -n 4 KEYS "*TEST*"` in non-tty?
and `sonic-db-cli` for the similar command?

Reply:
sorry, should use echo , results are the same , \n is taken as string.

```
admin@ASW-7005:~$ echo $(redis-cli -n 4 KEYS "*TEST*")
TEST1\n TEST2\n TEST3\/n
admin@ASW-7005:~$ echo "$(redis-cli -n 4 KEYS "*TEST*")"
TEST1\n
TEST2\n
TEST3\/n
admin@ASW-7005:~$ echo $(sonic-db-cli CONFIG_DB KEYS "*TEST*")
TEST1\n TEST2\n TEST3\/n
admin@ASW-7005:~$ echo "$(sonic-db-cli CONFIG_DB KEYS "*TEST*")"
TEST1\n
TEST2\n
TEST3\/n
```

Reply:
updated, able to go head and merge